code/modules/shuttle/on_move.dm

```diff
@@ -325,6 +337,7 @@ All ShuttleMove procs go here
 	var/knockdown = movement_force["KNOCKDOWN"]
 	if(knockdown)
 		Paralyze(knockdown)
+		trip_with_gun()
```

Comment:
for humans, directly adding misfire code to shuttle movement isn't something i'm comfortable with. couldn't it be a result of knockdown / paralysis generally instead of caused, specifically, by slipping and shuttle movement? in addition to resulting in cleaner code, i think it would feel more consistent.

that should be done by adding the trip_with_gun() call to /datum/status_effect/incapacitating/knockdown/on_apply() and /datum/status_effect/incapacitating/paralyzed/on_apply(), respectively (in case of paralysis, it should come BEFORE the traits are added, as TRAIT_HANDS_BLOCKED will cause items to drop. i considered pushing it even further down, to TRAIT_FLOORED, but that wouldn't be right, as TRAIT_FLOORED is also used when, for example, buckling people to beds)

Reply:
Proc has been moved out and now triggers on knockdown and paralyzed like you suggested.